### PR TITLE
API Skeleton

### PR DESCRIPTION
**Pull request by kanafghan**
Please notice, that this PR is based on #15.

### CODE REVIEW

**Comment by kanafghan** [comment]
@toin0u: Would you please also review this?

File: README.md [hunk 1]
```diff
@@ -149,5 +149,26 @@ app = webapp.WSGIApplication([
 ```
 
 
+## API Development
+The Fiziq backend provides a number of API endpoints. These endpoints are documented [here](doc/api/README.md).
+The implementation of these endpoints are located in the `src/api` folder. You can use the Google APIs Explorer
+tool in order to test the API endpoints of Fiziq. Its possible both locally (on the development machine) and
```
Comment (toin0u) [inline README.md]: Typo: It's

File: src/app.yaml [hunk 2]
```diff
@@ -12,6 +12,8 @@ handlers:
   upload: favicon\.ico
 - url: /tests.*
   script: lib.gaeunit.gaeunit.tester
+- url: /_ah/spi/.*
```
Comment (toin0u) [inline src/app.yaml:15]: typo: `api` I guess :)

File: README.md (hunk 1 above)
Comment (kanafghan) [inline README.md]: fixed

File: src/app.yaml (hunk 2 above)
Comment (kanafghan) [inline src/app.yaml:15]: no, this one is OK: https://cloud.google.com/appengine/docs/python/endpoints/api_server

Comment (toin0u) [inline src/app.yaml:15]: ah funny! @_@

**Comment by toin0u** [comment]
:+1:

**Pull request merged**